src/Platforms/PostgreSQLPlatform.php

```diff
@@ -1047,15 +1067,21 @@ public function getGuidTypeDeclarationSQL(array $column)
      */
     public function getDateTimeTypeDeclarationSQL(array $column)
     {
-        return 'TIMESTAMP(0) WITHOUT TIME ZONE';
+        $precision = $column['precision'] ?? 0;
+        assert(is_int($precision));
+
+        return sprintf('TIMESTAMP(%d) WITHOUT TIME ZONE', $precision);
```

Comment:
I guess precision will be always unsigned:

```suggestion
        return sprintf('TIMESTAMP(%u) WITHOUT TIME ZONE', $precision);
```